lib/locales/en.ts

```diff
@@ -350,6 +356,7 @@ export const locale: Locale = {
       'The Dedication of the Cathedral Basilica of Saint-Denis, France',
     dedication_of_the_cathedral_of_notre_dame_de_strasbourg_france:
       'The Dedication of the Cathedral of Notre-Dame of Strasbourg',
+    dedication_of_the_cathedral_saint_john_of_lyon_france: 'The Dedication of the Primatiale Cathedral',
```

Comment:
1. Rename the key to `dedication_of_the_cathedral_of_saint_john_of_lyon_france` (add missing `of_` before `saint`.
2. I believe that `Primatiale Cathedral` is a mistake. While the Cathedral is a Primatiale Cathedral, I think we should change the the English key value to `The Dedication of the Cathedral of Saint John of Lyon`.
3. According to [Wikipedia](https://en.wikipedia.org/wiki/Lyon_Cathedral), the Cathedral is dedicated to Saint John the Baptist, not Saint John of Lyon (if there is any such saint), therefore we would rename this key (and `en.ts` value) accordingly (see the suggestion below).

```suggestion
    dedication_of_the_cathedral_of_saint_john_the_baptist_lyon_france: 'The Dedication of the Cathedral of Saint John the Baptist, Lyon',
```